modules/comments/comments.php

```diff
@@ -272,7 +272,19 @@ public function comment_form_after() {
 		<div id="respond" class="comment-respond">
 			<h3 id="reply-title" class="comment-reply-title"><?php comment_form_title( esc_html( $params['greeting'] ), esc_html( $params['greeting_reply'] ) ); ?> <small><?php cancel_comment_reply_link( esc_html__( 'Cancel reply' , 'jetpack') ); ?></small></h3>
 			<div id="commentform" class="comment-form">
-				<iframe src="<?php echo esc_url( $url ); ?>" allowtransparency="<?php echo $transparent; ?>" style="width:100%; height: <?php echo $height; ?>px;border:0px;" frameBorder="0" scrolling="no" name="jetpack_remote_comment" id="jetpack_remote_comment"></iframe>
+				<iframe src="<?php echo esc_url( $url ); ?>" style="width:100%; height: <?php echo $height; ?>px; border:0; overflow:hidden;" name="jetpack_remote_comment" class="jetpack_remote_comment" id="jetpack_remote_comment"></iframe>			
+				<!--[if !IE]><!-->
+				<script>
+				r(function(){
+					//document.getElementById("jetpack_remote_comment").allowTransparency = <?php echo $transparent; ?>;
+					var comment_form_elements = document.getElementsByClassName("jetpack_remote_comment");
+					for (var i = 0; i < comment_form_elements.length; i++) {
+					    comment_form_elements[i].allowTransparency = <?php echo $transparent; ?>;
```

Comment:
@shahthepro after adding your change in a test site, the form flickers when it slides down as can be seen in this video https://cloudup.com/cizAOCNEbct
This is noticeable whether user is logged in or out.
After this line that adds the transparency, I added
```js
comment_form_elements[i].scrolling = 'no';
```
and it solved the flickering.

Also, can you format this following [JavaScript Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/)? Please add spacing, use single quotes for string, use meaningful names.

Finally, this JavaScript block declares `r()` with global scope and could conflict with something so please wrap this in a closure.

Thanks!